.github/workflows/sv-tests-ci.yml

```diff
@@ -39,11 +39,16 @@ jobs:
             deps: autoconf autotools-dev bison default-jre flex libfl-dev cmake pkg-config tclsh uuid-dev
             runners_filter: UhdmVerilator
           - name: yosys
-            deps: bison clang tcl-dev flex libfl-dev
+            deps: bison clang tcl-dev flex libfl-dev pkg-config libreadline-dev
+            runners_filter: Yosys
```

Comment:
With changed name of ``vanilla-yosys-uhdm-plugin`` binary, this is no longer needed.

Reply:
Removed